src/contractor.jl

```diff
@@ -70,6 +78,36 @@ end
 
 (C::Contractor{N,1,F1,F2,ex})(A::Interval{T}, X::IntervalBox{N,T}) where {N,F1,F2,ex,T} = C(IntervalBox(A), X)
 
+
+function (C::BasicContractor{F1,F2})(
```

Comment:
Ah, I forgot that you can't define this for abstract types :/  You can use metaprogramming instead to avoid this code duplication.

Reply:
Actually the easier thing would just be to factor out the contents of the function into a new function.